synchronizer/connectors/jira.py

```diff
@@ -62,6 +64,35 @@ def export_worklogs(self, worklogs):
                 print(err)
                 raise self.ExportException(i)
 
+    def validate_worklogs(self, worklogs):
+        """
+        Check existing worklog's task id in target resource
+        """
+        for worklog in worklogs:
+            try:
+                data = {
+                    'started': self.convert_datetime(worklog.date_started),
+                    'timeSpentSeconds': self.round_seconds(
+                        worklog.duration
+                    ),
+                    'comment': worklog.comment
+                }
+
+                self._get(
+                    'issue/{0}/worklog'.format(worklog.issue_id),
+                    data,
+                    params={'notifyUsers': 'false'}
+                )
+            except WrongIssueIDException:

```

Comment:
This feature is not from this issue. Should not be merged with this pull request.